src/nrc/templates/admin/base_site.html

```diff
@@ -39,3 +39,14 @@ <h1 id="site-name"><a href="{% url 'admin:index' %}">{{ settings.PROJECT_NAME }}
     {% endif %}
     {{ block.super }}
 {% endblock %}
+
+{% block footer %}
+    <div id="footer">
+        {% url 'admin:login' as admin_login_url %}
+        {% if request.path != admin_login_url %}
+            <div class="version" title="Git SHA: {{ settings.GIT_SHA|default:'' }}">
+                {% blocktrans with version=settings.RELEASE %}version {{ version }}{% endblocktrans %}
+            </div>
+        {% endif %}
+    </div>
+{% endblock %}
```

Comment:
Maybe this is something that should be part of open-api-framework, @annashamray thoughts?

Reply:
Similar question with the `ENVIRONMENT_SHOWN_IN_ADMIN` in the template. https://github.com/maykinmedia/referentielijsten/pull/14#discussion_r1630941769

Reply:
@stevenbal I agree, it looks like it belongs to open-api-framework